cli/render.go

```diff
@@ -282,8 +282,9 @@ func printCurrentFieldsInfo(fields []*report.Field) {
 
 		fmt.Print(strings.Repeat("  ", int(counter+1)))
 
-		for ; counter%field.Size != 0; counter++ {
+		for ; counter%field.Size != 0 {
```

Comment:
There is no need for `;` because there is no [initialization or postcondition](https://golangdocs.com/for-loop-in-golang) in this loop.